src/handlers/cloudBackup.ts

```diff
@@ -100,6 +106,7 @@ export async function encryptAndSaveDataToCloud(data: any, password: any, filena
       scope,
       sourcePath: sourceUri,
       targetPath: destinationPath,
+      update: true,
```

Comment:
this is dependent on https://github.com/rainbow-me/react-native-cloud-fs/pull/20